.eslintrc.json

```diff
@@ -37,10 +37,7 @@
             2,
             "all"
         ],
-        "linebreak-style": [
-            2,
-            "unix"
-        ],
+        "linebreak-style": "off",
```

Comment:
Why this change?

Reply:
because I use Windows and this caused me an error when running the tests, here I read how to correct it.

https://eslint.org/docs/rules/linebreak-style

However, I can revert to the previous configuration if they want to maintain that rule.
:)

Reply:
I see. By default git will commit the change as an LF instead of CRLF, so it should be fine 😄